roles/nf-core/templates/site.config.j2

```diff
@@ -22,21 +24,22 @@ process {
         time = 36.h
   }
 
-  {% endif %}
 }
 
-singularity.cacheDir = '{{ container_dir }}'
-
-{% if pipeline == 'rnaseq' %}
+params.reference_database = '{{ ngi_containers }}/{{ pipeline }}/{{ silva_zip }}'
 
-params.reverseStranded = true
+  {% endif %}
 
 {% endif %}
 
-{% if pipeline=='ampliseq' %}
+singularity.cacheDir = '{{ container_dir }}'
+singularity.libraryDir = '{{ container_dir }}'
 
-params.reference_database = '{{ ngi_containers }}/{{ pipeline }}/{{ silva_zip }}'
+params.seq_center = "SciLifeLab National Genomics Infrastructure {{ site_full | capitalize }}"
```

Comment:
(Oh my, this template was hard to read when showing changes 😅)
I think it would make sense to put stuff that applies to all pipelines at the start or end of the file.